cpp/log/etcd_consistent_store-inl.h

```diff
@@ -18,7 +18,7 @@ namespace {
 // etcd path constants.
 const char kClusterConfigFile[] = "/cluster_config";
 const char kUnsequencedDir[] = "/unsequenced/";
-const char kSequencedDir[] = "/sequenced/";
+const char kSequenceFile[] = "/sequencing";
```

Comment:
"mapping" could be anything, I think it needs to have some reference to "sequence" in there, I'll compromise with "/sequence_mapping" :)